### DISCUSSION TITLE
Splitting work between different nodes.

### DISCUSSION THREAD

**Discussion by osher** [OptimalBits/bull#1968] [Q&A]
Hi.

I'm new to `bull`, I'm coming from `kue`, which declares `bull` as it's successor.

However, in `kue` I could use a node as a producer and other nodes as consummers.
I tried to do the same with `bull`, but got errors for missing job processors.

What am I missing?
How can I do the same with `bull`?

**Comment by osher** [comment]
anyone?

**Comment by manast** [comment]
Why don't you provide code that is not working so we can correct it for you?
